tdms/include/vertex_phasors.h

```diff
@@ -0,0 +1,105 @@
+/**
+ * @file vertex_phasors.h
+ * @brief Contains a class that handles the complex amplitude extraction at the vertices.
+ */
+#pragma once
+
+#include <complex>
+
+#include "arrays.h"
+#include "field.h"
+#include "grid_labels.h"
+
+/**
+ * Class container for handling complex amplitude samples at the vertices, and their extraction.
+ *
+ * Abbreviated to CAmpSample in MATLAB code
+ */
+class VertexPhasors {
```

Comment:
I gave this an afternoon when developing, and concluded no.

Despite extracting the same thing; the method by which this happens, the data structures we need to setup to store the phasors, and the information that we need to take from the input file to know what we're doing, are considerably different in both cases. 

The only common functionality I could find was the correspondence between the `surface_EH{r,i}` and `camplitudes{R,I}` variables, and even these are different shapes since the `vertex_phasors` class does not always extract all the field components whilst the `surface_phasors` class appears to do so.

That's not to say it's _impossible_ for us to do this, just I don't think it's _worth_ us doing this. At least whilst we're this side of #70 (as MATLAB gone -> C++ native structures everywhere -> might open up common functionality / data storage options)

Reply:
Gottit. Fine by me.